src/user-guide/writing-workflows/parameterized-tasks.rst

```diff
@@ -545,31 +563,46 @@ point:
                model<chunk=4>[-P1Y] => model<chunk=1>
            """
 
-.. note::
+The inter-cycle trigger connects the first chunk in each cycle point to the
+last chunk in the previous cycle point. However, in this particular case it
+might be simpler to use a 3-monthly datetime cycle instead:
+
+.. code-block:: cylc
 
-   The inter-cycle trigger connects the first chunk in each cycle point
-   to the last chunk in the previous cycle point. Of course it would be simpler
-   to just use 3-monthly cycling:
+   [scheduling]
+       initial cycle point = 2020-01
+       [[graph]]
+           P3M = "model[-P3M] => model"
 
-   .. code-block:: cylc
 
-      [scheduling]
-          initial cycle point = 2020-01
-          [[graph]]
-              P3M = "model[-P3M] => model"
+For another example, here task ``model`` generates 10 files in sequence at it
+runs. Task ``proc_file0`` triggers when the model starts running, to wait for
+and process the first file; when that is done, ``proc_file1`` triggers to wait
+for the second file; and so on.
 
-Here's a possible valid use-case for mixed cycling: consider a portable
-date-time cycling workflow of model jobs that can each take too long to run on
-some supported platforms. This could be handled without changing the cycling
-structure of the workflow by splitting the run (at each cycle point) into a
-variable number of shorter steps, using more steps on less powerful hosts.
+.. code-block:: cylc
+
+   [task parameters]
+       file = 0..9
+   [scheduling]
+       initial cycle point = 2020-01
+       [[graph]]
+           P1Y = """
+               model:start => proc<file=0>
+               proc<file-1> => proc<file>
+               proc<file=9> => upload_products
+           """
+   [runtime]
+      [[model]]
+      [[proc<file>]]
+      [[upload_products]]
```

Comment:
Is there any point including these empty runtime sections?

Reply:
To show use of the parameter under runtime as well as in the graph. But I can remove the other two sections, to be consistent with previous examples.